env.example

```diff
@@ -0,0 +1 @@
+BASE_RPC_URL=
```

Comment:
forgot the dot in the file name? i.e. `.env.exmaple`

Reply:
Fixed in c3ec33557797777563f4854696fcdc94c4e4756d